invisible_cities/sierpe/fee.py

```diff
@@ -29,7 +29,7 @@
 f_LPF1 =  3 * units.MHZ
 f_LPF2 = 10 * units.MHZ
 ADC_TO_PES_LPF = 24.1  # After LPF, comes out from spe area
-ADC_TO_PES     = 23.1
+ADC_TO_PES     = 18.71
```

Comment:
I guess the consequences of changing these numbers are controled, right?

Reply:
I'd add the plots if I could (and I could find them at the moment) but the changes were checked by removing the variation and checking the first non pedestal peak before the 1pe fluctuation was added. 18.71 is the average position over the 12 pmts. Adding the dispersion after this change gives positions consistent with the DB gains.